cmd/kedge/main.go

```diff
@@ -334,8 +334,6 @@ func debugServer(logEntry *log.Entry, middlewares chi.Middlewares, noAuthMiddlew
 	m.Handle("/debug/events", middlewares.HandlerFunc(trace.Events))
 
 	return &http.Server{
-		WriteTimeout: *flagHttpMaxWriteTimeout,
```

Comment:
I wonder whether these flags are used anymore. If no, add a comment to the flag that they are deprecated.

Reply:
they are